server/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/Decision.java

```diff
@@ -64,14 +64,11 @@ public static Decision readFrom(StreamInput in) throws IOException {
             final String label = in.readOptionalString();
             final String explanation = in.readOptionalString();
             if (label == null && explanation == null) {
-                switch (type) {
-                    case YES:
-                        return YES;
-                    case THROTTLE:
-                        return THROTTLE;
-                    case NO:
-                        return NO;
-                }
+                return switch (type) {
+                    case YES -> YES;
+                    case THROTTLE -> THROTTLE;
+                    case NO -> NO;
```

Comment:
I had to check the code to make sure this was correct 😆

Reply:
I was also surprised by this and had to double check :smile: I guess such snippets is the reason why some code bases block static imports :smile: